backends/bmv2/CMakeLists.txt

```diff
@@ -177,7 +177,7 @@ set (XFAIL_TESTS
   testdata/p4_16_samples/psa-example-counters-bmv2.p4
   testdata/p4_16_samples/psa-example-digest-bmv2.p4
   testdata/p4_16_samples/psa-example-register2-bmv2.p4
-  testdata/p4_16_samples/psa-fwd-bmv2.p4
```

Comment:
removed, but the intermediate file is still updated?

Reply:
removed from XFAIL -- this test now passes...